src/proxy.rs

```diff
@@ -0,0 +1,23 @@
+//! Proxy services of validator
+//!
+
+use crate::deploy::Host;
+use anyhow::{Context, Result};
+use std::process::Command;
+
+/// Proxy RPC service
+pub fn rpc(host: &Host, local_port: u16) -> Result<()> {
+    let address = host.ipv4_address;
+    let user = &host.install_ssh_user;
+    println!("open up proxy at: http://localhost:{local_port:}");
```

Comment:
Potentially it would be also good to tell people how to close it again:

```suggestion
    println!("The Near RPC api of {host.name:} is now reachable at http://localhost:{local_port:}");
    println!("Press Ctrl-C to close it");
```